src/content/docs/vulnerability-management/integrations/aws.mdx

```diff
@@ -29,8 +31,57 @@ https://security-ingest-processor.service.newrelic.com/v1/security/webhooks/awss
 Select **POST** as the HTTP method.
 Select **Create a new connection**.
 
-## 2. Create a new connection
+### B. Create a new connection
 1. Fill in the fields in the new prompt.
 2. For **Destination** select **Other**.
 3. For **Authorization type** select **API Key**.
 4. For **API Key Name** type `Api-Key` and paste your New Relic License Key[license key](/docs/accounts-partnerships/install-new-relic/account-setup/license-key) as the value.
+
+## 2. Create an EventBridge rule
+
+Once you've created an API destination for ingest, create an Eventbridge rule to forward security-related events to an API destination for New Relic's ingest.
+
+### A. Define rule detail
+
+1. In your AWS UI navigate to **EventBridge > Rules > Create Rule**
```

Comment:
```suggestion
1. In your AWS UI go to **EventBridge > Rules > Create Rule**
```

Reply:
afaik Austin wants us to err on the side of go over navigate, nitpicky but just calling it out